Warning for encodedOffset

While running Komondor, I get this warning message in the logs : 

> ... PackageConfig/Sources/PackageConfig/DynamicLibraries.swift:46:37: warning: 'encodedOffset' is deprecated: encodedOffset has been deprecated as most common usage is incorrect. Use utf16Offset(in:) to achieve the same behavior.
>                                 return String($0.prefix(comment.encodedOffset))

Komondor v 1.0.4
PackageConfig v 0.13.0
Apple Swift version 5.3.1 (swiftlang-1200.0.39 clang-1200.0.32.6)
Target: x86_64-apple-darwin19.6.0

--- a/Sources/PackageConfig/DynamicLibraries.swift
+++ b/Sources/PackageConfig/DynamicLibraries.swift
@@ -43,7 +43,7 @@ enum DynamicLibraries {
 			.map(String.init)
 			.map {
 				guard let comment = $0.range(of: "//")?.lowerBound else { return $0 }
-				return String($0.prefix(comment.encodedOffset))
+				return String($0[..<comment])
 			}
 			.joined()
 			.replacingOccurrences(of: "\t", with: "")

